_posts/2020-10-27-wildfly-operator-0.4.1-released.adoc

```diff
@@ -0,0 +1,355 @@
+
+[source,bash]
+----
+$ oc new-project wildfly-demo
+----
+
+Then we install a `CatalogSource` to get access to the WildFly Operator that is in the operatorhub.io catalog:
+
+.operatorhub-io-catalogsource.yaml
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: operatorhub-io
+spec:
+  displayName: Operators from operatorhub.io
+  image: quay.io/operatorhubio/catalog:latest
+  sourceType: grpc
+----
+
+We install it by running:
+
+[source,bash]
+----
+$ oc apply -f operatorhub-io-catalogsource.yaml
+catalogsource.operators.coreos.com/operatorhub-io created
+----
+
+Then we create a subscription for the current version (0.4.1) of the WildFly Operator:
+
+.wildfly.yaml
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: wildfly
+  labels:
+    operators.coreos.com/wildfly.wildfly-demo: ''
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: wildfly
+  source: operatorhub-io
+  sourceNamespace: wildfly-demo
+  startingCSV: wildfly-operator.v0.4.1
+----
+
+Again, we install it on OpenShift by running:
+
+[source,bash]
+----
+$ oc apply -f wildfly.yaml
+subscription.operators.coreos.com/wildfly created
+----
+
+We will then wait until the WildFly Operator is installed by monitoring its installation:
+
+[source,bash]
+----
+$ oc get csv -w
+NAME                      DISPLAY   VERSION   REPLACES   PHASE
+wildfly-operator.v0.4.1   WildFly   0.4.1
+wildfly-operator.v0.4.1   WildFly   0.4.1                Pending
+...
+wildfly-operator.v0.4.1   WildFly   0.4.1                InstallReady
+...
+wildfly-operator.v0.4.1   WildFly   0.4.1                Installing
+...
+wildfly-operator.v0.4.1   WildFly   0.4.1                Succeeded
+----
+
+[NOTE]
+====
+For the purpose of this example, the WildFly Operator is only be installed in the current `wildfly-demo` namespace.
+All resources that are created must also be installed in that same namespace.
+====
+
+## Create an ImageStream
+
+Before we can deploy our application on OpenShift using the WildFly Operator, we will first create
+an `ImageStream` named `wildfly-demo-app` that will contain a stream of all our application images
+
+[source,bash]
+----
+$ oc import-image wildfly-demo-app:step-1          \
+    --from quay.io/jmesnil/wildfly-demo-app:step-1 \
+    --confirm
+----
+
+We have added the `wildfly-demo-app:step-1` imagestream tag to this imagestream by importing the Docker image from `quay.io/jmesnil/wildfly-demo-app:step-1`.
+
+Then we tag it with the `latest` tag that will be referenced from our deployments.
+
+[source,bash]
+----
+$ oc tag wildfly-demo-app:step-1  wildfly-demo-app:latest
+----
+
+At this point, we have an `ImageStreamTag` `wildfly-demo-app:latest` that we can use to deploy our application and upgrade it later seamlessly.
+
+We have finally the image inside OpenShift so we can deploy our application it.
```

Comment:
'deploy our application it.' - there is either an extra word here or missing something :-)